<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportForm</class>
 <widget class="QWidget" name="ReportForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stampe</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ordine</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="ordineAlfabeticoBox">
        <property name="text">
         <string>Alfabetico</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ordinePrezzoBox">
        <property name="text">
         <string>Per Prezzo</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Suddivisione</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="repartiBox">
        <property name="text">
         <string>Reparti</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="destinazioneBox">
        <property name="text">
         <string>Destinazione stampa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="noSuddivisioneBox">
        <property name="text">
         <string>Nessuna</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>destinazioneBox</zorder>
     <zorder>noSuddivisioneBox</zorder>
     <zorder>repartiBox</zorder>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="anteprimaBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Anteprima</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="esportaBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Esporta PDF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="chiudeBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Chiude</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>chiudeBtn</sender>
   <signal>clicked()</signal>
   <receiver>ReportForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>-2</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
